source/adapters/hip/usm.cpp

```diff
@@ -70,7 +70,6 @@ UR_APIEXPORT ur_result_t UR_APICALL USMFreeImpl(ur_context_handle_t hContext,
                                                 void *pMem) {
   ur_result_t Result = UR_RESULT_SUCCESS;
   try {
-    ScopedContext Active(hContext->getDevice());
```

Comment:
This changes has caused `hContext` to become unused. This warnign becomes an error when `UR_DEVELOPER_MODE=ON`.